lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1028,15 +1039,27 @@ export class UserAgentApplication {
 
             const correlationIdParam = `client-request-id=${requestCorrelationId}`;
 
-            const postLogoutQueryParam = this.getPostLogoutRedirectUri()
-                ? `&post_logout_redirect_uri=${encodeURIComponent(this.getPostLogoutRedirectUri())}`
-                : "";
+            let postLogoutQueryParam: string;
+            if (this.getPostLogoutRedirectUri()) {
+                postLogoutQueryParam = `&post_logout_redirect_uri=${encodeURIComponent(this.getPostLogoutRedirectUri())}`;
+                this.logger.verbose("redirectUri found and set");
+            } else {
+                postLogoutQueryParam = "";
+                this.logger.verbose("No redirectUri set for app. postLogoutQueryParam is empty");
+            }
 
-            const urlNavigate = this.authorityInstance.EndSessionEndpoint
-                ? `${this.authorityInstance.EndSessionEndpoint}?${correlationIdParam}${postLogoutQueryParam}`
-                : `${this.authority}oauth2/v2.0/logout?${correlationIdParam}${postLogoutQueryParam}`;
+            let urlNavigate: string;
+            if (this.authorityInstance.EndSessionEndpoint) {
+                urlNavigate = `${this.authorityInstance.EndSessionEndpoint}?${correlationIdParam}${postLogoutQueryParam}`;
+                this.logger.verbose(`EndSessionEndpoint found, urlNavigate set to: ${this.authorityInstance.EndSessionEndpoint}`);
```

Comment:
Not sure if Tenant ID is considered PII or not, but if so this url may contain PII

Reply:
Sure. I will make it Pii just in case.

Reply:
Yeah, lets treat anything that main contain data provided by the user to be PII (e.g. client id, tenant id, etc).

Reply:
And be sure to include non-pii log messages alongside any pii log messages.

Reply:
Yes, will do!